src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -1407,6 +1416,25 @@ var record = dcScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(
         assertThat(record.getErrors().toString(), containsString(Error.UNKNOWN_PROJECT.name()));
     }
 
+    @Test
+    void shouldScrapeWiseflowAndInsperaIdentifiers(){
+        var insperaIdentifier = "no.ntnu:inspera:22222:22222";
+        var wiseFlowIdentifier = "no.usn:wiseflow:11111:11111";
```

Comment:
Are these in fact URNs?

Reply:
It is urn yes, should we import it as string or something else? The plan is to place them in additional identifiers